src/main/groovy/ai/digital/integration/server/tasks/worker/SyncServerPluginsWithWorkerTask.groovy

```diff
@@ -0,0 +1,69 @@
+package ai.digital.integration.server.tasks.worker
+
+import ai.digital.integration.server.domain.Worker
+import ai.digital.integration.server.util.ProcessUtil
+import ai.digital.integration.server.util.ServerUtil
+import ai.digital.integration.server.util.WorkerUtil
+import org.apache.commons.io.FileUtils
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+
+import java.nio.file.Paths
+
+import static ai.digital.integration.server.constant.PluginConstant.PLUGIN_GROUP
+
+class SyncServerPluginsWithWorkerTask extends DefaultTask {
+    static NAME = "syncServerPluginsWithWorker"
+
+    def dependencies = [
+            DownloadAndExtractWorkerDistTask.NAME
+    ]
+
+    SyncServerPluginsWithWorkerTask() {
+        this.configure { ->
+            group = PLUGIN_GROUP
+            dependsOn(dependencies)
+        }
+    }
+
+    @TaskAction
+    def copyToWorkers() {
+        WorkerUtil.getWorkers(project).forEach { Worker worker ->
+            copyServerDirToWorkerDir(worker)
+        }
+    }
+
+    void copyServerDirToWorkerDir(Worker worker) {
+        if (!WorkerUtil.isExternalRuntimeWorker(worker, project)) {
+            def sourceDir = ServerUtil.getServerWorkingDir(project)
+            def destinationDir = Paths.get(WorkerUtil.getWorkerWorkingDir(worker, project)).toFile()
+            destinationDir.setExecutable(true)
```

Comment:
It's not clear for me why full folder has to be executable.
I think it has to be only files from `bin` folder

Reply:
removed that line, and next line for chmod is executing only on bin folder